### DISCUSSION TITLE
MSCS%40NWU/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#68] [Announcements]
# MSCS%40NWU/

https://csmsapp.github.io/MSCS%40NWU/

**Comment by beckysx** [comment]
美本3.7 无实习 2科研 录取了， 最后选择uci mcs，西北就业一般

**Comment by Jinx2000** [comment]
报个dp, 23fall 美本3.7 无实习无科研 一年TA, 无奖录, 应该决定要去了

**Reply by XuefengSong1207** [reply]
您好， 请问一下西北mscs research机会好找吗，以及转博士友好吗？感谢！！！

**Reply by AnnabelleMin** [reply]
西北科研机会比较多

**Reply by XuefengSong1207** [reply]
您有了解博士的上岸率吗，感谢！！！

**Comment by Paper-blue** [comment]
陆本末9，23fall，无实习无科研，无奖offer
但是考虑转博操作可能性，先defer一年等24fall再去

**Reply by charlesyangxinye** [reply]
兄弟 啥叫转博操作可能性defer啊？是拿到offer然后gap一年补科研背景是吗？

**Reply by Paper-blue** [reply]
是这样的，我的科研经历很匮乏，计划去当RA然后出个论文补一下这方面的背景

**Reply by charlesyangxinye** [reply]
哦哦了解了。我是本科有一点科研经历，发了两个水论文，这种的话去读ms之后然后联系导师进组做实验打工之后转博的机会怎么样呢？这个ms转phd是只要导师要就可以是吗？

**Reply by Paper-blue** [reply]
我还在准备阶段，没有实际操作过，所以也不清楚实际操作可能性大小，但是一般本校转博是比申外校友好些的，NU的handbook里的意思是老师同意接受就可以，所以跟老师搞好关系表现出学术能力应该是可行的

还有之前和23fall的PhD同学聊天，他说今年的PhD有两三个是ms转的，考虑到班级规模大小（一个项目加一块也就五十来人？）以及有的同学会选择先RA一年后PhD，NU转博的机会应该相对别的学校还是蛮高的了

**Reply by charlesyangxinye** [reply]
哦哦好谢谢哥。“有的同学会选择先RA一年后PhD”意思是在美ms读完之后做RA一年然后申PhD嘛？

**Reply by Paper-blue** [reply]
有的老师可能觉得某个学生有潜力但是需要挖掘，或者是看中了但是今年收的人已经满了
这种情况老师会问学生愿不愿意做一年RA，然后保证给同学明年的PhD位置，当然也有老师把学生鸽了的情况，也有风险

**Reply by charlesyangxinye** [reply]
okok了解了 谢谢兄弟

**Reply by Revive-dontwanttocode** [reply]
能不能求个bg呀

**Reply by xichenpan** [reply]
@Revive-dontwanttocode 可以看下datapoints里的有无 https://opencs.app/datapoints

**Reply by Revive-dontwanttocode** [reply]
手动@xichenpan 感谢~都忘了hhhh

**Reply by XuefengSong1207** [reply]
您好， 请问一下西北mscs research机会好找吗，以及转博士友好吗？感谢！！！

**Comment by Geeksongs** [comment]
1.修正一下这个项目，这个项目只是偏向于美本，除了美本的其他海本都是黑洞，24 fall ucsd gpa3.7 无科研实习我认识录了6个。
2.再报个24 fall，澳洲top3的dp，GPA4.0/4.0, rank 1/298，university medal, First-Class Honour 毕业，均分94，大厂实习一段+3科研，被拒。

**Reply by Rona-dt** [reply]
是不是overqualified了...我23fall是非美本的海本，bg比你差一些也录了😂

**Reply by Paper-blue** [reply]
绝对是oq吧，这背景这么强

**Reply by Geeksongs** [reply]
不一定是oq，美帝对澳洲学校不太待见，去年也有学长差不多背景的申请美国被全聚德。

**Reply by AnnabelleMin** [reply]
同ucsd录nwu cs感觉ucsd申研真的还不错。虽然感觉没人会去

**Reply by chenliangqi123** [reply]
大佬 我想问一下澳洲top3的那个dp 有没有读honor 他是三年本科申请美硕被拒吗

**Comment by shockylove** [comment]
英国 3.85 无科研 实习一段 无奖offer

**Comment by guapisolo** [comment]
24fall这项目画风突变，对陆本就业选手非常友好。补个 dp，UESTC cs，3.9|91+|rk3%，T107(S26)，无g，国奖一个。XCPC区域赛金四枚，没进wf。阿里数据库底层开发实习一段，几乎无科研。

**Reply by zqh0421** [reply]
感觉看了好几个ad都是有拿过果酱的

**Reply by Rockdu** [reply]
虽然好几个ad是拿果酱的，但不要忘了有rej是拿了3个果酱的（

**Reply by RainJamesY** [reply]
24fall第一轮被re，也有果酱；无论如何第一轮确实纯陆本黑洞

**Comment by SihanC29** [comment]
美本3.9+ cs&stat和info sci的双学位 无gre 只交了2实习 2无发表科研 r2录的 同录uiuc mcs 之前一直想走ds/金工 现在感觉还是先正统cs后再转 不知道咋选

**Reply by shockylove** [reply]
我也是西北+uiuc mcs录取，感觉就是科研走西北，招工走uiuc

**Reply by Snackiller** [reply]
学长你好，请问西北MSCS找实习情况怎么样

**Comment by Ashley0703** [comment]
在这贡献一个录取dp:
两财一贸金融本科(本科学校绩点比较拉）
ucb交换一年选的全是cs课 接近满绩 一门课的小ta
gatech 3d construction 暑研 一般科研推
nyush data mining方向ra 强推
科研均无发表产出，第二段比较solid，无实习

应该是海外交流经历和科研经历比较加分

**Comment by trtrtz** [comment]
这个项目真的是A*吗，和哥大mscs怎么选

**Reply by RainJamesY** [reply]
找工cu 读博nu

**Comment by Snackiller** [comment]
请问有没有这个项目的真实找工情况啊

**Comment by Aimbot6324** [comment]
26fall第一轮，有顶会、4pub被拒，千万千万不要申第一轮

**Reply by EthanJiang02** [reply]
难绷 gpa多少啊

**Reply by Aimbot6324** [reply]
3.6，很多都是在投估计也看不上。只能说这学校不像网传的那样重视科研，还是加州好

**Reply by dsyt-SichenGuo** [reply]
我两篇顶刊顶会分别是一作和共一第二都是在投西北第一轮被录了，但我觉得可能是美国暑研老板推荐信发力了，感觉西北的看的比较全面，绩点科研和美国con必须都得有，只有一个方面特别突出还是陆本就直接拒。不过楼主你肯定有更好的去处

**Comment by Rapct** [comment]
第一轮录取了，但是打听一圈是个很没用项目啊，找工回国都不行。
有点绝望捏，有没有别的学校啊。

**Reply by dsyt-SichenGuo** [reply]
也不至于吧，北美找工不太行，但是回国前辈跟我说认可度很高都能当华五硕用啊，他us第6 qs42回国不可能不认的，只是本身比较偏科研导向不好美国找工好像

**Reply by Rapct** [reply]
兄弟你好，我申请结束了，upenn se； 西北 cs； ucsd cs75； ucl information security； 爱丁堡 cs。 我不清楚北美两年后就业形势，感觉找工无望，可能要去ucl保排名了。祝好。

**Comment by Yeyuruoying11** [comment]
英本3.84 无实习 毕设机械臂，选修了研究生自动巡航小车项目。
